src/twisted/conch/ssh/filetransfer.py

```diff
@@ -58,7 +61,7 @@ def dataReceived(self, data):
             try:
                 f(data)
             except Exception:
-                log.err()
+                self.log.failure()
```

Comment:
[`Logger.failure()`](https://twistedmatrix.com/documents/current/api/twisted.logger.Logger.html#failure) requires a `format` argument. Perhaps:

```suggestion
                self.log.failure("Failed to handle packet of type {packetType}", packetType=packetType)
```